KDIC/BlogViewController.swift

```diff
@@ -1,100 +1,35 @@
-//
-//  BlogViewController.swift
-//  KDIC
-//
-//  Created by Shaun Mataire on 12/27/15.
-//  Copyright © 2015 Colin Tremblay. All rights reserved.
-//  NOTE: XCode wont let me refector this class is actully for the Blog
-
 import UIKit
 import SafariServices
 
-class BlogViewController: UIViewController,  UITableViewDataSource, UITableViewDelegate {
-
+class BlogViewController: UIViewController {
+    
     @IBOutlet weak var blogPostTableView: UITableView!
     
-    var blogsDict: NSArray?
+    var articles = [Article]()
+    
+    var kdicScrapperReqs: KDICScrapperReqs?
     
-    var kdicScrapperReqs: KdicScrapperReqs?
+    var blogJsonUrl = "https://kdic-scrapper.herokuapp.com/blogs"
     
     @IBOutlet weak var blogLoadActivityIndicator: UIActivityIndicatorView!
     override func viewDidLoad() {
         
-        //make request for json dictionary
-        self.kdicScrapperReqs = KdicScrapperReqs(urlStr: "https://kdic-scrapper.herokuapp.com/blogs", tableView: blogPostTableView, completionCall: getDataArray)
-        //kdicScrapperReqs.makeApiCall()
-        self.kdicScrapperReqs!.get_data_from_url()
+        self.kdicScrapperReqs = KDICScrapperReqs(urlStr: blogJsonUrl, tableView: blogPostTableView, completionCall: getDataArray)
```

Comment:
See my comments in KDICScrapperReqs class. Much of this class will be simplified.